app/install/kubernetes.md

```diff
@@ -14,11 +14,13 @@ links:
   gh_repo: "https://github.com/Kong/kong-dist-kubernetes/"
 ---
 
-<div class="alert alert-warning">
-  This documentation covers installing Kong
-  in Kubernetes. Kong can also be installed and used as a <a href="https://konghq.com/blog/kubernetes-ingress-controller-for-kong/">Kubernetes
-  Ingress Controller</a>.
-</div>
+# Kubernetes Ingress Controller
+
+You can install Kong CE or EE using our official <a href="https://github.com/Kong/kubernetes-ingress-controller">Kubernetes Ingress Controller</a>.
+
+You can learn more by checking out the <a href="https://github.com/Kong/kubernetes-ingress-controller/blob/master/README.md">README.md</a> file in the GitHub repository. To get up and running quickly, you can also follow the <a href="https://github.com/Kong/kubernetes-ingress-controller/tree/master/deploy">Minikube and Minishift tutorials</a> on the same repository.
+
+If you are curious, you can also read the <a href="https://konghq.com/blog/kubernetes-ingress-controller-for-kong/">official announcment</a> on our Blog. Finally, if you experience any problem or have some feedback, please open a new issue on GitHub.
```

Comment:
Could/should we link to the GitHub repo issues page?

Reply:
Updated with link.